src/bundle/Resources/config/services/controllers.yaml

```diff
@@ -1,7 +1,7 @@
 services:
     EzSystems\EzPlatformAdminUiBundle\Controller\:
         resource: "../../Controller/*"
-        exclude: "../../Controller/{Controller}"
+        exclude: "../../Controller/{FieldDefinitionController.php}"
```

Comment:
TBH I don't like this workaround. I'd rather go for expanding this definition into specific services. It feels like explicitness is more clear and understandable.

This is remark rather for the future, for now let's leave as-is, since it was given as one of the options.

cc @Nattfarinn

Reply:
Agreed, but the PR is big as it is, and from what I've seen I'd need to add most of the classes that reside in the subdirectory, which is around ~30 I believe?